### PR TITLE
Document various monitoring and alerting services

### PR DESCRIPTION
**Pull request by mcoffey-mt**
Documents on how contributors to our repository can get set up with the monitoring and alerting services. These also contain information on why someone might use one of the services.

### CODE REVIEW

**Review by chubberlisk — APPROVED** [review]
Very nice, appreciate the effort gone into this to document! Just lots of smol comments  and stuff.

File: README.md [hunk 1]
```diff
@@ -231,6 +231,7 @@ Note, this will only specifically enable the `RequestLogger`.
 
 ## Further documentation
 
+- [Monitoring and Alerting](/docs/monitoring-and-alerting/README.md)
```
[the remark this answers is not in the document]
Comment (mcoffey-mt) [reply README.md:234]: Done! :)

File: docs/monitoring-and-alerting/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+The HMPPS Integration API utilises numerous tools to assist identification and resolution of issues that may arise. The 
+purpose of these documents is to provide instruction on the use of each tool, how to set up an account with them, and 
+examples of where each may be useful. This ensures contributors are well-equipped to debug issues.
+
+## Services
+- [Azure Application Insights](azure-application-insights.md): Access metrics
+- [Kubernetes](kubernetes.md): Pod specific application logs
+- [Kibana](kibana.md): Kubernetes application logs
+- [Sentry](sentry.md): Error logs
+- [Grafana](grafana.md): Performance metrics
```
[the remark this answers is not in the document]
Comment (mcoffey-mt) [reply docs/monitoring-and-alerting/README.md:11]: Doneee!

File: docs/monitoring-and-alerting/kibana.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+[Kubernetes Logs](kubernetes.md). Kibana also allows time based filtering of logs.
+
+The information you'll find within the Kibana logs is any information that has been logged to the standard output of each
+pod.
+
+## Setup
+Follow Cloud Platform's [documentation](https://user-guide.cloud-platform.service.justice.gov.uk/documentation/logging-an-app/access-logs.html#accessing-application-log-data)
+
+## Steps
+[hmpps-integration-api-dashboard](https://kibana.cloud-platform.service.justice.gov.uk/_plugin/kibana/app/dashboards#/view/87dccad0-e503-11ed-a7a3-8d8f1c5a54a7?_g=(filters%3A!()%2CrefreshInterval%3A(pause%3A!t%2Cvalue%3A0)%2Ctime%3A(from%3Anow-1h%2Cto%3Anow)) is a direct link to the dashboard which the steps below lead you to.
```
[the remark this answers is not in the document]
Comment (mcoffey-mt) [reply docs/monitoring-and-alerting/kibana.md:14]: How strange, it displays correctly in Intellij. Swapped it for a short link to the same thing.

File: docs/monitoring-and-alerting/sentry.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+## Setup
+Access to Sentry is provided all members of the Ministry of Justice GitHub organization as outlined in their [documentation](https://operations-engineering.service.justice.gov.uk/documentation/services/sentry.html#sentry-io). 
+
+1. Once you have access, log in to the [Projects Dashboard](https://ministryofjustice.sentry.io/projects/)
+2. Click `Join a Team`
+3. Search for `hmpps-integration-api` 
+4. Join the team
+
+## Steps
```
[the remark this answers is not in the document]
Comment (mcoffey-mt) [reply docs/monitoring-and-alerting/sentry.md:17]: I didn't add a usecase to this one as it's just really straight forward to use. The same with steps, there's just not much to it :D

**Pull request merged**